### ISSUE TITLE
figure 4, seven configurations.

### ISSUE THREAD

**Issue by reshnashrestha** [ubsuny/23-Homework7G3#5]
FYI
![image](https://github.com/ubsuny/23-Homework7G3/assets/143649458/d79cbe13-ec8f-41e8-bd8e-05690ecbb5ee)

Fig. 4. The lowest energy configurations found for the cluster size of four
ion pairs along with their respective binding energies ~eV/ion pair!.

**Comment by reshnashrestha** [comment]
Have you chosen configurations?

**Comment by poojashresthacode** [comment]
@reshnashrestha there are 7 figures but the configuration naming are five, Can you enlighten on this?

**Comment by poojashresthacode** [comment]
i mean to say binding energy in place of configuration naming.

**Comment by poojashresthacode** [comment]
Does that mean that they have same binding energy?

**Comment by reshnashrestha** [comment]
Yeah they are same

**Comment by Pranjal-Srivastava-2023** [comment]
Yes, all these figures have the same binding energy around -28 units.
